content/en/docs/languages/python/getting-started.md

```diff
@@ -21,7 +21,7 @@ Ensure that you have the following installed locally:
 ## Example Application
 
 The following example uses a basic [Flask](https://flask.palletsprojects.com/)
-application. If you are not using Flask, that's OK — you can use OpenTelemetry
+application. If you have not used Flask before or prefer a different framework, that's OK — you can use OpenTelemetry
```

Comment:
I think this changes the meaning of this sentence. It was intended to mean "if you are not a flask user (bit django, etc.) you still can follow this tutorial, or you can follow along with your preferred framework, we have it for you", this changes it "if you do not have experience with flask, that's no problem, you still can follow this tutorial, ..."

I think both meanings are justified, maybe we can tune it accordingly.

Reply:
Thank you Severin, 

I see what you mean. If you would like to adjust it to be closer to the first meaning that is fine with me.

One way to potentially thread both might be "If you are not a Flask user, that's OK - you can still use OpenTelemetry and you can still follow this tutorial."

Reply:
yeah, that might be better. @open-telemetry/docs-approvers any thoughts on this?